src/ui/zcl_abapgit_gui_page_repo_sett.clas.abap

```diff
@@ -341,14 +341,17 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_SETT IMPLEMENTATION.
     READ TABLE it_post_fields INTO ls_post_field WITH KEY name = 'ignore_files'.
     ASSERT sy-subrc = 0.
 
-    SPLIT ls_post_field-value AT zif_abapgit_definitions=>c_newline INTO TABLE lt_ignore.
-    DELETE lt_ignore WHERE table_line IS INITIAL.
     " Remove everything
-    LOOP AT lo_dot->get_data( )-ignore INTO lv_ignore.
+    lt_ignore = lo_dot->get_data( )-ignore.
+    LOOP AT lt_ignore INTO lv_ignore.
       lo_dot->remove_ignore( iv_path = ''
                              iv_filename = lv_ignore ).
     ENDLOOP.
+
     " Add newly entered files
+    CLEAR lt_ignore.
+    SPLIT ls_post_field-value AT zif_abapgit_definitions=>c_newline INTO TABLE lt_ignore.
+    DELETE lt_ignore WHERE table_line IS INITIAL.
```

Comment:
this is something new, not related to downport?

Reply:
the logic is just moved down, I use the declared table instead of creating another one to buffer the method call

Reply:
ahh